recipes/repository.rb

```diff
@@ -37,6 +37,7 @@
     distribution distribution
     components components
     action :add
+    retries 4
```

Comment:
Could you make the number of retries configurable? (a default of `4` sounds good)

Reply:
Yes I'll add a parameter